### ISSUE TITLE
EAS Fastlane failing

### ISSUE THREAD

**Issue by Wolfleader101** [expo/eas-cli#1674]
### Build/Submit details page URL

https://expo.dev/accounts/wolfleader101/projects/pot-pirate/builds/22ee2a4d-2a2c-426d-bfde-57cd742e04e1

### Summary

It is failing in Run Fastlane at the end with the following error. This error only started occurring as of yesterday night.

```
› Executing react-native Pods/FBReactNativeSpec » [CP-User] Generate Specs
    Run script build phase 'Start Packager' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'PotPirate' from project 'PotPirate')
    Run script build phase 'Bundle React Native code and images' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'PotPirate' from project 'PotPirate')
    Run script build phase '[CP-User] Generate app.config for prebuilt Constants.manifest' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'EXConstants' from project 'Pods')
▸ ** ARCHIVE FAILED **
▸ The following build commands failed:
▸     PhaseScriptExecution [CP-User]\ Generate\ Specs /Users/expo/Library/Developer/Xcode/DerivedData/PotPirate-byuqhcbphskqumejjvyrwsnwvpze/Build/Intermediates.noindex/ArchiveIntermediates/PotPirate/IntermediateBuildFilesPath/Pods.build/Debug-iphoneos/FBReactNativeSpec.build/Script-46EB2E00015E30.sh (in target 'FBReactNativeSpec' from project 'Pods')
▸ (1 failure)
** ARCHIVE FAILED **
The following build commands failed:
    PhaseScriptExecution [CP-User]\ Generate\ Specs /Users/expo/Library/Developer/Xcode/DerivedData/PotPirate-byuqhcbphskqumejjvyrwsnwvpze/Build/Intermediates.noindex/ArchiveIntermediates/PotPirate/IntermediateBuildFilesPath/Pods.build/Debug-iphoneos/FBReactNativeSpec.build/Script-46EB2E00015E30.sh (in target 'FBReactNativeSpec' from project 'Pods')
(1 failure)
Exit status: 65
```

### Managed or bare?

Managed

### Environment

  expo-env-info 1.0.5 environment info:
    System:
      OS: Windows 10 10.0.22621
    Binaries:
      Node: 16.3.0 - C:\Program Files\nodejs\node.EXE
      Yarn: 1.22.19 - C:\Program Files\nodejs\yarn.CMD
      npm: 7.15.1 - C:\Program Files\nodejs\npm.CMD
    SDKs:
      Android SDK:
        API Levels: 28, 29, 30
        Build Tools: 28.0.3, 29.0.2, 29.0.3, 30.0.2, 30.0.3, 31.0.0
        System Images: android-29 | Google APIs Intel x86 Atom, android-30 | Google APIs Intel x86 Atom
    IDEs:
      Android Studio: Version     2020.3.0.0 AI-203.7717.56.2031.7678000
    Expo Workflow: managed

Expo Doctor:
🎉 Didn't find any issues with the project!

### Reproducible demo or steps to reproduce from a blank project

List of my depedencies
```json
  "dependencies": {
    "@acme/api": "^0.1.0",
    "@acme/tailwind-config": "^0.1.0",
    "@config-plugins/react-native-ble-plx": "^5.0.0",
    "@expo/vector-icons": "^13.0.0",
    "@react-native-community/masked-view": "^0.1.11",
    "@react-navigation/bottom-tabs": "^6.5.4",
    "@react-navigation/native": "^6.1.3",
    "@react-navigation/native-stack": "^6.9.9",
    "@shopify/flash-list": "1.3.1",
    "@tanstack/react-query": "^4.24.4",
    "@trpc/client": "^10.10.0",
    "@trpc/react-query": "^10.10.0",
    "@trpc/server": "^10.10.0",
    "clsx": "^1.2.1",
    "expo": "~47.0.13",
    "expo-dev-client": "~2.0.1",
    "expo-location": "~15.0.1",
    "expo-splash-screen": "~0.17.5",
    "expo-status-bar": "~1.4.2",
    "nativewind": "^2.0.11",
    "react": "18.1.0",
    "react-dom": "18.1.0",
    "react-native": "0.70.5",
    "react-native-auth0": "^2.17.0",
    "react-native-ble-plx": "^2.0.3",
    "react-native-gesture-handler": "~2.8.0",
    "react-native-maps": "1.3.2",
    "react-native-modal": "^13.0.1",
    "react-native-reanimated": "~2.12.0",
    "react-native-safe-area-context": "4.4.1",
    "react-native-screens": "~3.18.2"
  },```

**Comment by Wolfleader101** [comment]
Update:

Updating to react native 0.71.2 fixed the bug, however expo doctor will now complain about the react native version.

**Comment by brentvatne** [comment]
unable to reproduce this. I looked at your xcode logs from the build you linked to and saw this:

```
/Users/expo/workingdir/build/apps/expo/ios/Pods/../../../../node_modules/react-native/React/FBReactNativeSpec/../../scripts/react_native_pods_utils/script_phases.sh: line 24: syntax error near unexpected token `{
'
```

my first guess is this may be related to react-native support for pnpm workspaces. i believe this was fixed by our very own @byCedric in 0.71: https://github.com/facebook/react-native/pull/35430

you can try using patch-package to apply the same fix. we're working on getting react-native 0.71 support in sdk 48 for a release this quarter.

**Comment by Wolfleader101** [comment]
Thanks, 
Will look into using patch package.
